pylot/operator_creator.py

```diff
@@ -4,6 +4,7 @@
 # import Control operators.
 from pylot.control.pylot_agent_operator import PylotAgentOperator
 from pylot.control.ground_agent_operator import GroundAgentOperator
+from pylot.control.mpc.mpc_agent_operator import MPCAgentOperator
```

Comment:
We should move the import to inside the `create_mpc_agent_op` function.

Reply:
Done